presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -101,10 +103,15 @@ public void testShowCreateView()
     {
         onHive().executeQuery("DROP VIEW IF EXISTS hive_show_view");
 
-        onHive().executeQuery("CREATE VIEW hive_show_view AS SELECT * FROM nation");
+        onHive().executeQuery("CREATE VIEW hive_show_view AS SELECT n_nationkey FROM nation");
 
-        // view SQL depends on Hive distribution
-        assertThat(query("SHOW CREATE VIEW hive_show_view")).hasRowsCount(1);
+        QueryResult result = query("SHOW CREATE VIEW hive_show_view");
+
+        assertThat(result).hasRowsCount(1);
+        String actualSql = (String) result.row(0).get(0);
+        assertEquals(
+                actualSql.replaceAll("\n", " ").replaceAll("\\s+", " "),
```

Comment:
keeping `actualSql` intact is OK.

also, if we used to output nicely formatted result and in the future we stop doing that, i would like the test to tell me so.

```suggestion
                actualSql,
```